database/warehouse/replay/info.py

```diff
@@ -12,7 +12,8 @@
 
 class info(Injectable, Base):
     __tablename__ = "info"
-    __table_args__ = {"schema": "replay"}
+    __table_args__ = ( UniqueConstraint("filehash", name="filehash_unique")
```

Comment:
Here we enforce unique filehash for the replay. Consider adding index (or make primary_key)(?)